internal/log/log.go

```diff
@@ -36,6 +39,13 @@ const (
 	DebugLevel = slog.LevelDebug
 )
 
+func init() {
+	// Use the journal handler if stdout is not a TTY
+	if !term.IsTerminal(int(os.Stdout.Fd())) {
+		slog.SetDefault(slog.New(&JournalHandler{}))
+	}
```

Comment:
This applies to PAM too, right?

Probably we should not do it there though, or we won't honor the `disable_journal` option (that I find useful for testing).
And also for PAM we shuould first check for the `PAM_TTY` item (as done in `(m nativeModel) isTerminalTty()`, although I just noticed that PAM `initLogging` needs to be fixed too.

It's also something it's probably better not to do when building with `withexamplebroker`, that is the way tests do but also I personally use to do manual testing of the daemon.